apps/lockfile-explorer/src/shared.d.ts

```diff
@@ -0,0 +1 @@
+export * from '@rushstack/lockfile-explorer-web/lib/shared';
```

Comment:
Add a code comment explaining where this came frome:

We want the lflint CLI tool and the lockfile-explorer-web app.js bundle to share some code that has been bundled into shared.js.  This bundling happens in the lockfile-explorer-web project, which is not published to NPM, so we copy lockfile-explorer-web/dist/shared.js into lockfile-explorer/dist.  The shared.js/shared.d.ts stubs provide a way for TypeScript code to import these copied files with typings.

The shared.d.ts file reexports typings from lockfile-explorer-web.
The corresponding shared.js file reexports the real bundle from the "dist" folder.

Reply:
And then in shared.js we could have another comment that explains:

"This stub is used to import a bundle from lockfile-explorer-web without publishing that package to NPM. See shared.d.ts for details."

Reply:
added